client/src/containers/ProjectPeople/AddPeopleModal.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, memo, useEffect } from 'react';
+import { Modal, Button, Dropdown, DropdownOnSearchChangeData } from 'semantic-ui-react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import * as actions from './logic/actions';
+import { RootState } from 'typings/rootState';
+import { useTranslation } from 'react-i18next';
+
+interface Props {
+	project: WebApi.Entities.Projects;
+}
+
+const AddPeopleModal = ({ project, project: { lead, users: projectUsers } }: Props) => {
```

Comment:
it's better to destruct params in function body

Reply:
Comment accepted and implemented